### DISCUSSION TITLE
Versions no longer supported (EOL) of EJBCA

### DISCUSSION THREAD

**Discussion by Moobster** [Keyfactor/ejbca-ce#254] [Q&A]
Hello, 

I am trying to look everywhere for the versions of EJBCA that are no longer supported and I can't find any.

Does any of you know which versions are not supported anymore?

Thank you for your help.

Best,
Rhea

**Comment by primetomas** [comment]
Not sure what you mean, can you specify? There is no SLA support for Community. If you have issues, you should always upgrade to the latest released community version.
